indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -3977,14 +4048,25 @@ protected void emitNoticesQueueSize()
 
   protected void emitLag()
   {
-    if (spec.isSuspended() || !stateManager.isSteadyState()) {
+    log.info(stateManager.getSupervisorState().getBasicState().toString());
```

Comment:
Could the format of these logs be improved? Also, should they be info logs?

Reply:
@tejaswini-imply - these log lines have no context. They are unlikely to be useful.

Reply:
I have used them for debugging, pushed them by mistake. I'll revert these.